timeline/templates/timeline/timeline.html

```diff
@@ -65,9 +71,22 @@ <h3><i class="iconify" data-icon="mdi:clock-time-eight"></i> {{ timeline_title }
 {% block javascript %}
   {{ block.super }}
 
+  <!-- Settings for Javascript -->
+  <script type="text/javascript">
```

Comment:
This section is for the search results page and should not be included for the timeline view.

Reply:
But this section corresponds to pagination on client site. As it was mentioned in discussion there should be some way of making pagination on server side and then sorting options on client one. So I will change it